migrationtools/databases.py

```diff
@@ -121,6 +121,17 @@ def _setup_tables(self, meta):
         self.registration = meta.tables['registrations_registration']
         self.user = meta.tables['auth_user']
 
+    def get_source_from_authority(self, authority):
+        if authority == 'clinic':
+            name = 'CLINIC USSD App'
+        elif authority == 'personal':
+            name = 'PUBLIC USSD App'
+        elif authority == 'chw':
+            name = 'CHW USSD App'
```

Comment:
They are the only values that should be there (AFAIK) but I will add an else clause to guard against the unexpected.